src/Models/Nodes.php

```
@@ -0,0 +1,39 @@
+    public function current(): ?NodeInterface
+    {
+        /** @var null|NodeInterface $result */
+        return parent::current();
+    }
+
+    /**
+     * @param mixed $offset
+     *
+     * @return null|NodeInterface
+     */
+    public function offsetGet(mixed $offset): ?NodeInterface
```

PHPStan (2.1.15)

Return type (OpenFGA\Models\NodeInterface|null) of method OpenFGA\Models\Nodes::offsetGet() should be covariant with return type (OpenFGA\Models\Node|null) of method OpenFGA\Models\AbstractIndexedCollection<OpenFGA\Models\Node>::offsetGet()


(method.childReturnType)